checklists/alz_checklist.en.json

```diff
@@ -368,12 +365,11 @@
         {
             "category": "Resource Organization",
             "subcategory": "Naming and tagging",
-            "text": "It is recommended to follow Microsoft Best Practice Naming Standards",
+            "text": "Use a well defined naming scheme for resources, such as Microsoft Best Practice Naming Standards",
             "description": "Consider using the Azure naming tool available at https://aka.ms/azurenamingtool",
```

Comment:
For some entries like this, there is an additional "description" field. We should agree on uniform policy about the presence of this field, then add or remove across all the entries. In AVD checklist for example, "description" is used for all the entries.

Reply:
We can tackle that as part of a bigger overhaul - right now this is trying to address the open issues.  We are tracking it, but we need to get through this cycle first.